code/src/main/webapp/JSP/test.jsp

```diff
@@ -0,0 +1,29 @@
+
+
+<%@page language="java" contentType="text/html" pageEncoding="UTF-8" %>
```

Comment:
why do we have two test.jsp files? Are they exercising different paths or something?